client/src/Components/Location/Location.js

```diff
@@ -2,11 +2,47 @@ import React, { Component } from 'react';
 import './Location.css';
 
 class Location extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {wifi: '', privacy: '', quiet: '', foodAndDrink: ''};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({wifi: event.target.wifi});
+    this.setState({privacy: event.target.privacy});
+    this.setState({quiet: event.target.quiet});
+    this.setState({foodAndDrink: event.target.foodAndDrink});
+  }
+
+  handleSubmit(event) {
+    alert('A name was submitted: ' + this.state.coffee);
```

Comment:
What is this? `this.state.coffee` doesn't even seem to exist.

Either way, we should be notifying the user in a better way than an `alert`.